package.json

```diff
@@ -45,11 +45,12 @@
     "webpack-dev-server": "^3.11.2"
   },
   "dependencies": {
-    "bootstrap": "^5.0.2",
+    "bootstrap": "^4.6.0",
```

Comment:
Is this version downgrade intentional?

Reply:
Yes it is, we use `react-bootstrap` for some frontend styling and it currently doesn't work with bootstrap 5.0